scripts/create-docker-machine-and-run-it

```diff
@@ -0,0 +1,186 @@
+#!/bin/bash
+#
+# Setup a docker machine for development
+set -eu
+
+readonly PROGNAME=$(basename "$0")
+readonly ARGS=("$@")
+# Docker settings
+MACHINE_NAME="devel"
+HUGO_VERSION=0.20.2
+NODE_VERSION=6.10.0
```

Comment:
The only place necessary to edit the version numbers is this file. I think it's okay. Some of the entries from the template files are overwritten when the script runs. The node and hugo versions always get overwritten.